pkg/util/rand/rand.go

```diff
@@ -32,6 +33,21 @@ var rng = struct {
 	rand: rand.New(rand.NewSource(time.Now().UTC().UnixNano())),
 }
 
+// Intsn return a range of ints, from
+// [min, max]
+func Intsn(min int, n int, max int) []int {
```

Comment:
rand ports no longer needed, im gonna remove this.
I think we can test ports in order and preemptively avoid collisions by creating services beforehand.